saas-ibp-bootstrap/ordering-org-vars.yml

```diff
@@ -0,0 +1,17 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+api_endpoint: https://xxx
+api_authtype: ibmcloud
+api_key: xxx
+api_secret: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```

Comment:
These seem to be in several of the vars files. Are they common and if so, could they go in a `credentials-vars.yml` or the `common-vars.yml`?

Reply:
that is true - these came from the standard tutorial - and in theory each of the orgs and orderers could have different IBP consoles